src/NHibernate.Test/Legacy/FooBarTest.cs

```diff
@@ -2831,7 +2829,20 @@ public void PersistCollections()
 
 			// serialize and then deserialize the session.
 			Stream stream = new MemoryStream();
-			IFormatter formatter = new BinaryFormatter();
+#if NETFX
+			var formatter = new BinaryFormatter();
+#else
+			var selector = new SurrogateSelector();
+			selector.AddSurrogate(
```

Comment:
Somewhere deep in the `Baz` hierarchy some class has few properties of `CultureInfo` type. Unfortunately it is not marked as serializable in .NET Core / .NET Standard.